src/templates/use-cases.tsx

```diff
@@ -173,134 +118,145 @@ const components = {
   QuizWidget,
 }
 
-const Title = styled.h1`
-  font-size: 2.5rem;
-  font-weight: 700;
-  margin-top: 1rem;
-`
-
-const SummaryPoint = styled.li`
-  font-size: 1rem;
-  color: ${(props) => props.theme.colors.text300};
-  margin-bottom: 0rem;
-  line-height: auto;
-`
-
-const SummaryBox = styled.div``
-
-const StyledButtonDropdown = styled(ButtonDropdown)`
-  margin-bottom: 2rem;
-  display: flex;
-  justify-content: flex-end;
-  text-align: center;
-  @media (min-width: ${(props) => props.theme.breakpoints.s}) {
-    align-self: flex-end;
-  }
-`
-
-const StyledEmoji = styled(Emoji)`
-  margin-right: 1rem;
-  flex-shrink: 0;
-`
-
-const MobileButtonDropdown = styled(StyledButtonDropdown)`
-  margin-bottom: 0rem;
-  @media (min-width: ${(props) => props.theme.breakpoints.l}) {
-    display: none;
-  }
-`
-
-const Container = styled.div`
-  position: relative;
-`
-
-const HeroContainer = styled.div`
-  background: ${(props) => props.theme.colors.cardGradient};
-  box-shadow: inset 0px -1px 0px rgba(0, 0, 0, 0.1);
-  display: flex;
-  justify-content: flex-end;
-  max-height: 608px;
-  min-height: 608px;
-  width: 100%;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column-reverse;
-    max-height: 100%;
-  }
-`
-
-const Image = styled(GatsbyImage)<{ useCase: string }>`
-  flex: 1 1 100%;
-  background-size: cover;
-  background-repeat: no-repeat;
-  right: 0;
-  bottom: 0;
-  background-size: cover;
-  max-width: ${({ useCase }) =>
-    useCase === `dao` ? `572px` : useCase === `defi` ? `80%` : `640px`};
-  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
-    width: 100%;
-    height: 100%;
-    max-height: 340px;
-    max-width: ${({ useCase }) =>
-      useCase === "defi" ? "100%" : "min(405px, 98%)"};
-    overflow: initial;
-    align-self: center;
-    margin: 0;
-  }
-`
-
-const MoreContent = styled(Link)`
-  width: 100%;
-  background: ${(props) => props.theme.colors.ednBackground};
-  padding: 1rem;
-  display: flex;
-  justify-content: center;
-  &:hover {
-    background: ${(props) => props.theme.colors.background};
-  }
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    display: none;
-  }
-`
-
-const MobileTableOfContents = styled(TableOfContents)`
-  position: relative;
-  z-index: 2;
-`
-
-const StyledBannerNotification = styled(BannerNotification)`
-  display: flex;
-  justify-content: center;
-  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
-    display: none;
-  }
-`
+const HeroContainer = (props: ChildOnlyProp) => (
+  <Flex
+    bg="cardGradient"
+    boxShadow="inset 0px -1px 0px rgba(0, 0, 0, 0.1)"
+    flexDirection={{ base: "column-reverse", lg: "row" }}
+    justifyContent="flex-end"
+    minH="608px"
```

Comment:
I think we need to force this height. In prod we have this defined height:
![image](https://user-images.githubusercontent.com/468158/227349677-fb9057f9-1efc-4333-8bd9-ce8547fac108.png)


```suggestion
    h="608px"
```